reset_indexed_elements.py

```diff
@@ -0,0 +1,85 @@
+#
```

Comment:
@sBouzols , you told me that you tested it in a dry-run pod?

Reply:
I tested it locally with containers (study-server, directory-server) but not with an admin-tools container.
Maybe we should find a solution to avoid this mistake in the future. I'm looking for it.

Reply:
Ok add a way and some doc on README to manage to run a script in a container locally.
Will do a separate PR to refactor the codebase tree and allow to COPY all python scripts in the image for the futur.